docs/src/scalar_pds.md

```diff
@@ -0,0 +1,146 @@
+
+# [Example 1](@id example-1)
+
+We want to solve
+
+```math
+u' =  u^2 - u,\quad u(0) = 0.95
+```
+
+for ``0≤ t≤ 10``.
+Here,  we can choose ``p(u)=u^2`` as production term and ``d(u)=u`` as destruction term.
+The exact solution of this problem is
+
+```math
+u(t) = \frac{19}{19+e^{t}}.
+```
+
+Next, we show how to solve this scalar PDS in the way discussed above.
+Please note that must use [`PDSProblem`](@ref) to create the problem.
+Furthermore, we use static matrices and vectors from [StaticArrays.jl](https://juliaarrays.github.io/StaticArrays.jl/stable/) instead of standard arrays for efficiency.
+
+
+```@example example_1
+using PositiveIntegrators, StaticArrays, Plots
+
+u0 = @SVector [0.95] # 1-vector
+tspan = (0.0, 10.0)
+
+# Attention: Input u is a 1-vector
+prod(u, p, t) = @SMatrix [u[1]^2] # create static 1x1-matrix
+dest(u, p, t) = @SVector [u[1]] # create static 1-vector
+prob = PDSProblem(prod, dest, u0, tspan) 
+
+sol = solve(prob, MPRK22(1.0))
+
+# plot
+tt = collect(0:0.1:10)
+f(t) = 19.0 ./ (19.0 .+ exp.(t)) # exact solution
+plot(tt, f(tt), label="exact")
+plot!(sol, label="u")
+```
+
+# [Example 2] (@id example-2)
+
+Next, we want to compute positive solutions of a more challenging scalar PDS. 
+In [Example 1](@ref example-1) we could have also used standard schemes from [OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/) and use the solver option `isoutofdomain` to ensure positivity.
+But this is not always the case as the following example will show.
+
+We want to compute the nonnegative solution of 
+
+```math
+u'(t) = -\sqrt{\lvert u(t)\rvert },\quad u(0)=1. 
+```
+
+for ``t≥ 0``.
+Please note that this initial value problem has infinitely many solutions
+
+```math 
+u(t) = \begin{cases} \frac{1}{4}(t-2)^2, & 0≤ t< 2,\\ 0, & 2≤ t < t^*,\\ -\frac{1}{4}(t-2)^2, & t^*≤  t, \end{cases}
+```
+
+where ``t^*≥ 2`` is arbitrary.
+But among these, the only nonnegative solution is
+
+```math 
+u(t) = \begin{cases} \frac{1}{4}(t-2)^2, & 0≤ t< 2,\\ 0, & 2≤ t. \end{cases}
+```
+
+And it is this solution that we want to compute. 
+
+First, we try this using a standard solver from [OrdinaryDiffEq.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/).
+We try to enforce positivity with the solver option `isoutofdomain` by specifying that negative solution components are not acceptable.
+
+```@example
+using OrdinaryDiffEqRosenbrock
+
+tspan = (0.0, 3.0)
+u0 = 1.0
+
+f(u, p, t) = -sqrt(abs(u))
+prob = ODEProblem(f, u0, tspan)
+
+sol = solve(prob, Rosenbrock23(); isoutofdomain = (u, p, t) -> any(<(0), u))
+```
+
+We see that `isoutofdomain` cannot be used to ensure nonnegative solutions in this case, as the computation stops at about ``t≈ 2`` before the desired final time is reached. 
+At least for first and second order explicit Runge-Kutta schemes this can also be shown analytically. A short computation reveals that to ensure nonnegative solutions the time step size must tend to zero if the numerical solution tends to zero. 
```

Comment:
```suggestion
For at least first- and second-order explicit Runge-Kutta schemes, this can also be shown analytically. A brief computation reveals that to ensure nonnegative solutions, the time step size must tend to zero if the numerical solution tends to zero.
```

Is this published somewhere?